osh/hub/osh_xmlrpc/scan.py

```diff
@@ -237,18 +237,24 @@ def find_tasks(request, query):
      * 'nvr': search by specific NVR
      * 'regex': find by provided regex (this is not match, but find, if you
                 want match, change your regex to "^<regex>$")
+    Additionally, query supports following optional keys:
+     * 'comment': string, search by comment
+     * 'states': list, search by task states
+     * 'latest': boolean, specify whether to query only the latest task
```

Comment:
Latest is not optional, its value will be always set by the client.

Reply:
Thanks I realized it indeed has default value. This line is removed.